# HG changeset patch
# User John W. Eaton <jwe@octave.org>
# Date 1421955990 18000
#      Thu Jan 22 14:46:30 2015 -0500
# Branch gui-release
# Node ID 18c213fa9b20f502484bd620ab2556830e9b0683
# Parent  3156773fcc809f49cc12dafbb8cf3f463f5e282a
# Parent  7005044f3080e6f70650a4e29a9354df0b2b9ea2
maint: Periodic merge of stable to gui-release.

diff --git a/libinterp/corefcn/toplev.cc b/libinterp/corefcn/toplev.cc
--- a/libinterp/corefcn/toplev.cc
+++ b/libinterp/corefcn/toplev.cc
@@ -826,17 +826,17 @@ DEFUN (warranty, , ,
 @deftypefn {Built-in Function} {} warranty ()\n\
 Describe the conditions for copying and distributing Octave.\n\
 @end deftypefn")
 {
   octave_value_list retval;
 
   octave_stdout << "\n" << octave_name_version_and_copyright () << "\n\
 \n\
-GNU Octave free software; you can redistribute it and/or modify\n\
+GNU Octave is free software; you can redistribute it and/or modify\n\
 it under the terms of the GNU General Public License as published by\n\
 the Free Software Foundation; either version 3 of the License, or\n\
 (at your option) any later version.\n\
 \n\
 GNU Octave is distributed in the hope that it will be useful,\n\
 but WITHOUT ANY WARRANTY; without even the implied warranty of\n\
 MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the\n\
 GNU General Public License for more details.\n\
